.github/workflows/pylint.yml

```diff
@@ -17,6 +17,12 @@ jobs:
         conda-channels: 'conda-forge'
     - name: Install dependencies
       run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        pip install pandas
+        pip install requests
+        pip install validators
+        pip install opencv-python
         conda env update --file environment.yml
```

Comment:
Will this still take a long time with the conda update command still here?

Reply:
Doesn't take too long. Im pretty sure dependency resolution when building a new environment with conda is what takes up most of the time.